code/game/objects/structures/simple_doors.dm

```diff
@@ -240,3 +247,7 @@
 
 /obj/structure/simple_door/resin/New(var/newloc,var/material_name)
 	..(newloc, "resin")
+
+/obj/structure/simple_door/cult/New(var/newloc, var/material_name)
+	..(newloc, "cult")
+	color = "#4a3631" // looks better than the standard cult colours
```

Comment:
No one has used this colour before. Maybe add it to `color.dm` and use a define so peeps know what colour it is when reading? (If they're not using a text editor that displays it for you)